arches/app/models/graph.py

```diff
@@ -563,22 +563,18 @@ def delete(self):
                 )
             )
 
-    def delete_instances(self, verbose=False):
+    def delete_instances(self, verbose=False, request=None):
         """
         deletes all associated resource instances
 
         """
-        resources_to_delete = list(Resource.objects.filter(graph_id=self.graphid))
-        delete_count = len(resources_to_delete)
-        if verbose is True:
-            bar = pyprind.ProgBar(delete_count)
-        for resource in resources_to_delete:
-            resource.delete()
-            if verbose is True:
-                bar.update()
-        if verbose is True:
-            print(bar)
-        return delete_count
+        from arches.app.etl_modules.bulk_data_deletion import BulkDataDeletion
```

Comment:
Does it cause a circular import if you move this to the top of the file?

Reply:
fix in latest